### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by April0616**
6 user stories

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -58,16 +58,21 @@ Format: `help`
 === Adding a person: `add`
 
 Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` +
+Format: `add n/NAME m/MATRIC_NUMBER g/GENDER p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG] [tt/NUM_OF_MONTHS/TEMPORARY TAG]...` +
```
Comment (zacharytang) [inline docs/UserGuide.adoc]: Consider whether matric number and gender should be compulsory tags?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -149,27 +160,37 @@ Returns `john` and `John Doe`
...
+==== Deleting multiple persons
+
+Deletes the multiple specified persons from the address book. +
+Format: `delete [INDEX1], [INDEX2], [INDEX3]...` +
+Alternatives: `d` , `remove`
 
-Deletes the specified person from the address book. +
-Format: `delete INDEX` +
+Deletes the contacts which you haven't used the 'edit', 'photo' or 'find' command for K months. +
+Format: `delete old/K` +
```
Comment (zacharytang) [inline docs/UserGuide.adoc]: Perhaps could rename "K" to a more readable word

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -149,27 +160,37 @@ Returns `john` and `John Doe`
 * `find Betsy Tim John` +
 Returns any person having names `Betsy`, `Tim`, or `John`
 
-=== Deleting a person or a tag: `delete`
-==== Deleting a Person
+=== Deleting multiple persons or a tag: `delete`
+==== Deleting multiple persons
+
+Deletes the multiple specified persons from the address book. +
+Format: `delete [INDEX1], [INDEX2], [INDEX3]...` +
```
Comment (zacharytang) [inline docs/UserGuide.adoc]: What happens when `delete` is called without any parameters? This line implies that all parameters are optional

**Review by zacharytang — DISMISSED** [review]
Looks good! May want to read comments and share your opinions on some issues I've raised

File: docs/UserGuide.adoc (hunk 3 above)
Comment (April0616) [reply docs/UserGuide.adoc]: oh i see!

File: docs/UserGuide.adoc (hunk 2 above)
Comment (April0616) [reply docs/UserGuide.adoc]: oh yes

File: docs/UserGuide.adoc (hunk 1 above)
Comment (April0616) [reply docs/UserGuide.adoc]: it's a good point. I think gender is compulsory but not matric no. :)

Comment (April0616) [reply docs/UserGuide.adoc]: oh sry...I accidentally deleted ur ReviewMsg...is there any way to recover it?

**Review by nbriannl — APPROVED** [review]
Checkstyle errors showed up after deconflict with master. Clear the minor Checkstyle errors like trailing whitespaces. And we are good.

**Pull request merged**